pages/common/gimp.md

```diff
@@ -0,0 +1,24 @@
+# gimp
+
+> An image manipulation and paint program.
+> More information: <https://docs.gimp.org/en/gimp-fire-up.html#gimp-concepts-running-command-line>.
+
+- Don't show the splash screen:
+
+`gimp --no-splash`
+
+- Start a new instance, even if there is already a running one:
+
+`gimp --new-instance`
+
+- Open the given file as new image:
```

Comment:
```suggestion
- Create and open a new image at the specified path:
```

Reply:
This doesn't create a new file at the given path. It takes the given (existing) image and opens it as tough it wouldn't exist. Pretty much like a template.